src/shared/store.js

```diff
@@ -4,19 +4,19 @@ import {
     combineReducers,
     applyMiddleware,
 } from 'redux';
-import { routerReducer, routerMiddleware } from 'react-router-redux';
 import thunk from 'redux-thunk';
 /* eslint-disable import/no-extraneous-dependencies */
 import appEnhancers from '__app_modules__redux_enhancers__';
 import devEnhancers from './devTools';
 
 export function createRootReducer(reducers) {
-    return combineReducers({ ...reducers, routing: routerReducer });
+    if (Object.keys(reducers).length === 0) return state => state;
```

Comment:
@johangirod without that, I have an warning on each page, because the INIT action is executed before the reducer is replaced